src/components/Input/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import cn from "classnames";
+
+import styles from "./styles.module.scss";
+
+type IInputRadioProps = {
+  label: string;
+  type: "single" | "multiple";
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const Input: React.FC<IInputRadioProps> = React.memo(function Input({
```

Comment:
It's better to think about naming. Input sounds too abstract
Input can be text, radio, checkbox, range, date, etc.
I think the best practice would be to separate them off. RadioInput and CheckboxInput